app/assets/javascripts/modules/accordion-with-descriptions.js

```diff
@@ -122,6 +126,53 @@
         $node.attr("aria-expanded", state);
       }
 
+      function setSessionStorage() {
+        var isOpenSubsections = $('.subsection--is-open').length;
+        if (isOpenSubsections) {
+          var $openSubsections = $('.subsection--is-open');
+          $openSubsections.each(function(index) {
+            var subsectionOpenContentId = $(this).find('.subsection__content').attr('id');
+            sessionStorage.setItem( subsectionOpenContentId , subsectionOpenContentId);
+          });
+        }
+      }
+
+      function removeSessionStorage() {
```

Comment:
Actually - ignore me. This is in the context of the accordion isn't it. I think it makes sense just like you have it.